docs/buildah-copy.1.md

```diff
@@ -48,6 +48,11 @@ Build context directory. Specifying a context directory causes Buildah to
 chroot into the context directory. This means copying files pointed at
 by symbolic links outside of the chroot will fail.
 
+**--exclude** *pattern*
+
+Exclude copying files matching the specified pattern. Option can be specified
+multiple times.
```

Comment:
Perhaps a reference to the format of patterns being described in `containerignore(5)` from the `common` package?